packages/db/src/types/MemberRole.ts

```diff
@@ -0,0 +1,40 @@
+import { MemberRole } from "../public";
+
+export const MemberRoleRanking = {
+	[MemberRole.admin]: 0,
+	[MemberRole.editor]: 1,
+	[MemberRole.contributor]: 2,
+} as const;
+
+/**
+ * Compare two member roles
+ * @param a - The first member role
+ * @param operator - The operator to use
+ * @param b - The second member role
+ * @returns true if the operator is true for the two roles, false otherwise
+ */
+export const compareMemberRoles = (
```

Comment:
i like this implementation! would you mind consolidating the other roleRanking code (core/lib/authorization/rolesRanking.ts) with this? it uses the opposite numbering order so I don't think we should leave both around!

Reply:
ah yeah good idea! i was meaning to do that after reviewing your previous permissions related pr but then couldnt find it anymore haha